packages/auth/src/AuthProvider.tsx

```diff
@@ -212,6 +212,10 @@ export class AuthProvider extends React.Component<
   }
 
   logIn = async (options?: any) => {
+    this.setState({
+      ...this.state,
```

Comment:
I see that it's done this way in other places in this file, but this line really isn't needed. React will automatically do a shallow merge when updating the state.